server/common/HatoholArmPluginInterface.cc

```diff
@@ -157,6 +157,11 @@ struct HatoholArmPluginInterface::Impl {
 		connected = false;
 		hapi->onSessionChanged(NULL);
 	}
+	
+	bool getconnectedstatus(void) {
```

Comment:
Thank you for comments. I use the isConnected().
Because, return type of the method is bool.
(Function name is used many "is ~".)